receiver/README.md

```diff
@@ -1,11 +1,27 @@
-A variety of receivers are available to the OpenCensus Service (both Agent and Collector)
+**Note** This documentation is still in progress. For any questions, please reach out in the
+[OpenTelemetry Gitter](https://github.com/open-telemetry/community#Communication) or
+refer to the [issues page](https://github.com/open-telemetry/opentelemetry-service/issues).
 
-__Currently there are some inconsistencies between Agent and Collector configuration, those will be addressed by issue
-[#135](https://github.com/census-instrumentation/opencensus-service/issues/135).__ 
+# Receivers
+A receiver is how data gets into OpenTelemetry Service. Generally, a receiver accepts data in a specified format and can
+support [traces]() and/or [metrics]().
```

Comment:
If anyone has a good link that i can add here, please share!

Reply:
Consider https://github.com/open-telemetry/opentelemetry-proto/blob/2ccb7cb0cf038086955c46eebea767fddb331d16/src/opentelemetry/proto/trace/v1/trace.proto and https://github.com/open-telemetry/opentelemetry-proto/blob/2ccb7cb0cf038086955c46eebea767fddb331d16/src/opentelemetry/proto/metrics/v1/metrics.proto.

Reply:
Thank you! I'll start with those and we can iterate on explaining the data the svc receives in a future pr.